crutest/src/cli.rs

```diff
@@ -799,76 +817,130 @@ async fn process_cli_command(
                     "Internal write count buffer not filled".to_string(),
                 )))
                 .await
-            } else if ri.write_log.is_empty() {
+            } else if let Some(ri) = ri_option {
+                if ri.write_log.is_empty() {
+                    fw.send(CliMessage::Error(CrucibleError::GenericError(
+                        "Internal write count buffer empty".to_string(),
+                    )))
+                    .await
+                } else {
+                    let mut vec: Vec<u8> = vec![255; 2];
+                    vec[0] = (offset % 255) as u8;
+                    vec[1] = ri.write_log.get_seed(offset) % 255;
+                    fw.send(CliMessage::ExpectedResponse(offset, vec)).await
+                }
+            } else {
                 fw.send(CliMessage::Error(CrucibleError::GenericError(
-                    "Internal write count buffer empty".to_string(),
+                    "Info not initialized".to_string(),
                 )))
                 .await
-            } else {
-                let mut vec: Vec<u8> = vec![255; 2];
-                vec[0] = (offset % 255) as u8;
-                vec[1] = ri.write_log.get_seed(offset) % 255;
-                fw.send(CliMessage::ExpectedResponse(offset, vec)).await
             }
         }
         CliMessage::Export => {
-            if ri.write_log.is_empty() {
-                fw.send(CliMessage::Error(CrucibleError::GenericError(
-                    "Info not initialized".to_string(),
-                )))
-                .await
-            } else if let Some(vo) = verify_output {
-                println!("Exporting write history to {vo:?}");
-                match write_json(&vo, &ri.write_log, true) {
-                    Ok(_) => fw.send(CliMessage::DoneOk).await,
-                    Err(e) => {
-                        println!("Failed writing to {vo:?} with {e}");
-                        fw.send(CliMessage::Error(CrucibleError::GenericError(
-                            "Failed writing to file".to_string(),
-                        )))
-                        .await
+            if let Some(ri) = ri_option {
+                if ri.write_log.is_empty() {
+                    fw.send(CliMessage::Error(CrucibleError::GenericError(
+                        "Info not initialized".to_string(),
```

Comment:
Is this the correct error message?  It's confusing to me, because this message is used for both `write_log.is_empty()` and `ri_option.is_none()` cases.  Above, we use `"Internal write count buffer empty"` for the former.

(same question applies below in a bunch of places)

Reply:
It's not the correct message any longer, but even better, it's also not possible.
If we are at the place where we check for a write log, then it must exist, as we now have an `<Option<RegionInfo>>` instead of an "empty" one.  So, if the RegionInfo exists, then we have a write log and we can avoid this whole check.  I'll make that change in all the places where we check for it.